lua/tool_balance/tools/base/emitter.lua

```diff
@@ -0,0 +1,40 @@
+-- base/emitter
+
+-- config
+local config = { 
+    delay = { min = 0.5, max = math.huge },
+}
+
+-- min and max values for gmod_emitter
+local values = {
+    {}, --ply
+    {}, --key
+    config.delay
+}
+
+local clampMethod = cfcToolBalance.clampMethod 
+local clampFunction = cfcToolBalance.clampFunction
+local callAfter = cfcToolBalance.callAfter
+
+local function wrapEmitter()
+    MakeEmitter = clampFunction(MakeEmitter, values)
+    
+    local EMITTER =  scripted_ents.GetStored( "gmod_emitter" ).t
+    EMITTER.SetupDataTables = callAfter(EMITTER.SetupDataTables, function(self, ...)
```

Comment:
not sure this is the best way to be doing this. 
The issue was SetDelay is being defined after the emitter is placed and is a different function for each emitter.